core/dbt/task/run.py

```diff
@@ -170,9 +170,8 @@ def get_node_representation(self):
         return str(relation)
 
     def describe_node(self):
-        return "{} model {}".format(
-            self.node.get_materialization(), self.get_node_representation()
-        )
+        # TODO CL 'language' will be moved to node level when we change representation
+        return f"{self.node.config.get('language', 'sql')} {self.node.get_materialization()} model {self.get_node_representation()}"
```

Comment:
Should we skip if `sql` (implied), and include only if `python`? Especially since there will be many many adapters that don't support Python in the first cut.

I do like this as a way to cue people into the new capabilities, though. Multi-lingual dbt, it's here to be

Reply:
Either way works for me! Sounds like a product call!

Reply:
I would lean toward always indicating the language (since longer term it will probably head that way anyway?), but it's not a strongly held opinion!

Reply:
always it is :)